model/model.js

```diff
@@ -15,16 +28,29 @@ exports.selectAllArticles = (topic, sort_by = 'created_at', order = 'DESC') => {
 
 	let queryValues = [];
 	if (topic) {
+		const validTopics = ['cats', 'mitch', 'paper'];
```

Comment:
Although this works currently, what if as the production database grows there are more topics added? We could dynamically get the current topics in the database and make this the safe list rather than hardcoding it.